process_manager/tables.py

```diff
@@ -43,6 +54,23 @@ class Meta:  # noqa: D106
     select = tables.CheckBoxColumn(
         accessor="uuid",
         verbose_name="Select",
-        checked="checked",
-        attrs={"th__input": {"onclick": "toggle(this)"}},
+        attrs={
+            "th__input": {
+                "id": "header-checkbox",
+                "hx-preserve": "true",
+                "_": header_checkbox_hyperscript,
```

Comment:
What does this underscore represent?

Reply:
That is the html attribute that hyperscript uses to store its scripts on individual elements.